feed-integration-webapp/src/main/java/org/bahmni/module/feedintegration/atomfeed/client/AtomFeedProperties.java

```diff
@@ -0,0 +1,72 @@
+package org.bahmni.module.feedintegration.atomfeed.client;
+
+import java.io.InputStream;
+import java.util.Properties;
+
+public class AtomFeedProperties {
+
+
+    private static final String FEED_CONNECT_TIMEOUT = "feed.connectionTimeoutInMilliseconds";
+    private static final String FEED_REPLY_TIMEOUT = "feed.replyTimeoutInMilliseconds";
+    private static final String FEED_MAX_FAILED_EVENTS = "feed.maxFailedEvents";
+    private static final String FAILED_EVENT_MAX_RETRY = "feed.failedEventMaxRetry";
+
+    public static final String DEFAULT_PROPERTY_FILENAME = "/atomfeed.properties";
+
+    private Properties properties;
+
+    private static AtomFeedProperties atomFeedProperties;
+
+    private AtomFeedProperties() {
+        InputStream propertyStream = null;
+        try {
+            propertyStream = this.getClass().getResourceAsStream(DEFAULT_PROPERTY_FILENAME);
+            properties = new Properties();
+            properties.load(propertyStream);
+
+        } catch (Exception e) {
+        } finally {
+            if (null != propertyStream) {
```

Comment:
This line is not needed. Even if a null pointer exception is thrown in the next line, it will get caught and ignored.

Reply:
Line has been removed.